libcontainer/process_linux.go

```diff
@@ -22,6 +22,10 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// Synchronisation value for cgroup namespace setup.
+// The same constant is defined in nsexec.c as "CREATECGROUPNS".
+const createCgroupns byte = (1 << 7)
```

Comment:
I really would prefer if we represented both the same way. Either `0x80` or `1<<7` everywhere.